examples/ws/JSON/src/json/JSONParser.java

```diff
@@ -0,0 +1,18 @@
+
+package json;
+
+/**
+ * @Desc The main class for parsing a JSON via remote or local.
```

Comment:
@jashanj0tsingh -- never heard of `@Desc` javadoc tag. Remove (all places). Just keep the sentence itself.